mlflow/utils/uri.py

```diff
@@ -129,3 +129,7 @@ def _join_posixpaths_and_append_absolute_suffixes(prefix_path, suffix_path):
     # joined path
     suffix_path = suffix_path.lstrip(posixpath.sep)
     return posixpath.join(prefix_path, suffix_path)
+
+
+def is_artifact_acled_uri(artifact_uri):
```

Comment:
Nit: can we rename this to `is_databricks_acled_artifacts_uri`? This helps to clarify that we're checking for a piece of Databricks-specific functionality.

Reply:
Done.